### PR TITLE
Port: Include change of FreeBSD port

### PR DESCRIPTION
**Pull request by wwmayer**

### CODE REVIEW

File: src/Gui/SoFCCSysDragger.cpp [hunk 1]
```diff
@@ -96,7 +96,7 @@ TDragger::TDragger()
 {
     SO_KIT_CONSTRUCTOR(TDragger);
 
-#if defined(Q_OS_MACOS)
+#if defined(Q_OS_MAC) || defined(Q_OS_FREEBSD)
```
Comment (chennes) [inline src/Gui/SoFCCSysDragger.cpp:99]: The `Q_OS_MAC` define is a deprecated synonym for `Q_OS_DARWIN`.

File: src/Gui/SoFCCSysDragger.cpp
Comment (thyssentishman) [inline src/Gui/SoFCCSysDragger.cpp:1]: I just realized that adding `... || defined(__OpenBSD__)` here as well fixes #16392. Should I make a submit a separate PR for this or can it be merged with this?

File: src/Gui/SoFCCSysDragger.cpp (hunk 1 above)
Comment (wwmayer) [reply src/Gui/SoFCCSysDragger.cpp:99]: I've copied it from the FreeBSD port. I guess the define should still be `Q_OS_MACOS` as this is not marked as deprecated.

File: src/Gui/SoFCCSysDragger.cpp
Comment (thyssentishman) [reply src/Gui/SoFCCSysDragger.cpp:1]: Thank you :)
